oca_dependencies.txt

```diff
@@ -0,0 +1,3 @@
+web https://github.com/brainbeanapps/web 11.0-mig-web_widget_dropdown_dynamic
+server-ux
+connector-jira
```

Comment:
why are you adding connector-jira here?

Reply:
Unfortunately, `multi_step_wizard` lives there in v11 :(